docs/hoon/twig/cen-call/sig-open.md

```diff
@@ -13,6 +13,10 @@ sort: 3
 :rap(p :make(q +6 r))
 ```
 
+```
+=<(p %=(q +6 r))
+```
+
```

Comment:
Or even

=+  r=2
=+  ^=  q
    |_  a/@
    ++  p  +(a)
    --

So that ~(p q r) is =<(p q(+< r)). Though idk if conflating pqr the subtwig
positions and pqr the arm, wing, and twig referents respectively is a good
idea.

On Thursday, 23 June 2016, Anton Dyudin <antechno777@gmail.com> wrote:

> I think
>
> =+  q=|=(a/@ +(a))
> =<($ %=(q +6 2))
>
> should also work?
>
> On Thursday, 23 June 2016, Joe Bryan <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In docs/hoon/twig/cen-call/sig-open.md
>> <https://github.com/urbit/docs/pull/55#discussion_r68288438>:
>>
>> > @@ -13,6 +13,10 @@ sort: 3
>> >  :rap(p :make(q +6 r))
>> >  ```
>> >
>> > +```
>> > +=<(p %=(q +6 r))
>> > +```
>> > +
>>
>> This is another example of the ambiguity in the expansions. I puzzled
>> over this one for quite a while.
>>
>> I started with this
>>
>> %~($ |=(a/@ +(a)) 2)
>>
>> And then tried this
>>
>>  =<($ %=(|=(a/@ +(a)) +6 2))
>>
>> which doesn't compile, and can't even be typed in dojo. Eventually, I
>> reread the twig description, so that p is a wing, and got this to work:
>>
>>  =<($ =+(q=|=(a/@ +(a)) %=(q +6 2)))
>>
>> I thought about making the case for that expansions, till I saw the list
>> unwrapping in the ? twigs ...
>>
>> /cc @ohAitch <https://github.com/ohAitch>
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/urbit/docs/pull/55/files/fb89b530694cb031b6e33d0b360f304e6cb77089#r68288438>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ABxXhtSbSSFpSSlhLpdooWOGyzx98ylWks5qOs96gaJpZM4I9D6c>
>> .
>>
>